home.css

```diff
@@ -0,0 +1,613 @@
+body {
+  font-family: 'lato', sans-serif;
+  min-height: 100vh;
+  background-color: #e5e5e527;
+}
+
+.social-media,
+.navbar {
+  display: none;
+}
+
+.box {
+  background: url('./images/desktop.png') no-repeat;
+  background-size: cover;
+  margin-bottom: 0;
+  padding-bottom: 50px;
+}
+
+.hamburger {
+  border: none;
+  background: transparent;
+  position: relative;
+  margin-top: 8%;
+  padding-left: 5%;
+}
+
+.hamburger img {
+  width: 60%;
+  height: 70%;
+  margin-bottom: 80%;
+}
+
+.hello {
+  position: relative;
+  color: #f35826;
+  font-size: 200%;
+  margin-bottom: 0%;
+  padding-left: 5%;
+}
+
+.heading {
+  position: relative;
+  color: #f35826;
+  font-size: 400%;
+  margin-top: 0%;
+  font-weight: bold;
+  padding-left: 5%;
+}
+
+/* intro section */
+.intro {
+  border: 5px #fff solid;
+  margin-left: 5%;
+  font-size: 110%;
+  margin-right: 15%;
+  margin-top: 5%;
+  font-weight: normal;
+  opacity: 0.7;
+}
+
+.due-date {
+  position: relative;
+  font-weight: bold;
+  color: #585858;
+  font-size: 200%;
+  margin-left: 5%;
+  margin-bottom: 1%;
+  margin-top: 5%;
+}
+
+.venue {
+  color: #222121;
+  margin-left: 5%;
+  font-size: 110%;
+  margin-bottom: 5%;
+  opacity: 0.7;
+}
+
+/* Main program section */
+.program {
+  background: url('./images/main-program.jpg') no-repeat 0, 0 #0a0a0a;
+  background-size: cover;
+  margin-top: 0%;
+}
+
+.main-program {
+  position: relative;
+  color: #fff;
+  padding-top: 10%;
+  padding-bottom: 2%;
+  text-align: center;
+}
+
+.main-program::after {
+  content: '';
+  height: 2px;
+  width: 5%;
+  background: #f35826;
+  position: absolute;
+  bottom: 0;
+  left: 47%;
+}
+
+.activities {
+  padding-top: 5%;
+  list-style-type: none;
+  font-size: 90%;
+  opacity: 0.65;
+}
+
+.activity {
+  display: flex;
+  border: none;
+  background-color: #3c3f46;
+  gap: 10px;
+  margin: 2%;
+}
+
+.activities .activity:hover {
+  cursor: pointer;
+  position: relative;
+  border: #fff 2px solid;
+}
+
+.img-activity {
+  padding: 3%;
+  height: 120px;
+  margin-top: 3%;
+  width: 15%;
+}
+
+.description {
+  display: flex;
+  align-items: center;
+  color: #f35826;
+  font-weight: bold;
+  padding-right: 5%;
+  margin-right: 50px;
+  width: 10%;
+}
+
+.explanation {
+  font-size: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #fff;
+}
+
+.whole-program {
+  display: none;
+}
+
+.program-btn {
+  position: relative;
+  display: flex;
+  justify-content: center;
+}
+
+.join {
+  padding: 4%;
+  background-color: #f35826;
+  border: none;
+  color: #fff;
+  font-size: 140%;
+  margin: 5%;
+  margin-bottom: 10%;
+}
+
+.join:hover {
+  cursor: pointer;
+}
+
+.join:hover:hover {
+  background-color: #fff;
+  color: #f35826;
+}
+
+.join:hover:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+/* Featured speakers section */
+.speakers-section {
+  margin-top: 5%;
+}
+
+.featured-speakers {
+  text-align: center;
+  position: relative;
+  padding-bottom: 2%;
+}
```

Comment:
[OPTIONAL] It would be nicer to increase the gap between speakers to improve the UI design:

<img width="203" alt="image" src="https://user-images.githubusercontent.com/77190203/176998822-7ab48302-56d0-464c-97a2-86594edda029.png">